public/cantusstatic/templates/cantusstatic/default.html

```diff
@@ -0,0 +1,14 @@
+{% extends "empty_base.html" %}
+
+<!-- <title>{{ flatpage.title }}</title> -->
+
+{% block body %}
+<div class="container">
+    <div class="row">
+        <div class="col-lg-12">
+            <h2>{{ flatpage.title }}</h2>
+            <p>{{ flatpage.content }}</p>
```

Comment:
Is it the case that the content is always going to be one paragraph? It might be safer to put it in a `div`.